src/components/bind-stores-mixin.coffee

```diff
@@ -0,0 +1,28 @@
+_ = require 'underscore'
+
+module.exports =
+
+  # can modify which event you want to bind on as needed.
+  _defaultEvent: 'change'
+
+  _defaultUpdate: ->
+    @forceUpdate()
+
+  _addListener: (bind) ->
```

Comment:
Would it be clearer if this was `_addListener: ({store, listenTo, callback}) ->`?